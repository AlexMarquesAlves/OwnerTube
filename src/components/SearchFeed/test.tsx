import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import { SearchFeed } from ".";

it("should render SearchFeed with text Hello", () => {
  render(<SearchFeed>Hello</SearchFeed>);

  const heading = screen.getByRole("heading", { name: /Hello/ });

  expect(heading).toBeInTheDocument();
  expect(heading).toMatchSnapshot();
});
