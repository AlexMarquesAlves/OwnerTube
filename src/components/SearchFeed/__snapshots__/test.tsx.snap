// Vitest Snapshot v1

exports[`should render SearchFeed with text Hello 1`] = `
<h1>
  SearchFeed
  Hello
</h1>
`;
